libs/core/src/lib/carousel/carousel.service.ts

```diff
@@ -59,6 +59,19 @@ export class CarouselService implements OnDestroy {
     /** @hidden */
     private _currentTransitionPx = 0;
 
+    /**
+     * @hidden,
+     */
+    private _isRtl = false;
+
+    get isRtl(): boolean {
+        return this._isRtl;
+    }
+
+    set isRtl(isrtl: boolean) {
+        this._isRtl = isrtl;
+    }
```

Comment:
why do you need getter and setter in this case?

Reply:
fixed, using property isRtl.